x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -110,12 +111,30 @@ const fillForm = (wrapper: ReactWrapper) => {
   });
 };
 
+const fillFormReactTestingLib = async (renderResult: RenderResult) => {
+  const titleInput = within(renderResult.getByTestId('caseTitle')).getByTestId('input');
+  userEvent.type(titleInput, sampleData.title);
+
+  const descriptionInput = renderResult.container.querySelector(
+    `[data-test-subj="caseDescription"] textarea`
+  );
+  if (descriptionInput) {
+    userEvent.type(descriptionInput, sampleData.description);
+  }
+  const caseTags = renderResult.getByTestId('caseTags');
+  for (let i = 0; i < sampleTags.length; i++) {
+    const tagsInput = await within(caseTags).findByTestId('comboBoxInput');
+    userEvent.type(tagsInput, `${sampleTags[i]}{enter}`);
```

Comment:
Amazing! Wasn't aware of that :)

Reply:
this is only available with `userEvent` which is easier to use than `fireEvent` and already handles the `act` wrapping. That's why there's no "act" in most of of these, only on the ones that perform async code.